dmutils/forms/fields.py

```diff
@@ -17,7 +17,7 @@
 class StripWhitespaceStringField(StringField):
```

Comment:
So the suggestion now is that all of the implementation here should be moved into `dm_fields` classes; these classes would inherit from the `dm_fields` ones and would be deprecated.

Reply:
Do we want to stick a `DeprecationWarning` on them?

Reply:
Yeah, reckon a warning emitted by the constructor is the right thing here? And move any tests over from the old thing to the new thing.

Reply:
Done